kafka-client/src/main/java/dev/responsive/kafka/internal/db/rs3/RS3TableFactory.java

```diff
@@ -64,23 +80,47 @@ public RemoteKVTable<WalEntry> kvTable(
     );
   }
 
+  public RemoteWindowTable<WalEntry> windowTable(
+      final String storeName,
+      final Duration defaultTtl,
+      final ResponsiveMetrics responsiveMetrics,
+      final ResponsiveMetrics.MetricScopeBuilder scopeBuilder,
+      final Supplier<Integer> computeNumKafkaPartitions
+  ) {
+    final var rs3Client = connector.connect();
+    final UUID storeId = createdStores.computeIfAbsent(storeName, n -> createStore(
+        storeName,
+        CreateStoreTypes.StoreType.WINDOW,
+        Optional.of(ClockType.STREAM_TIME),
```

Comment:
just a heads up we still throw a ConfigErr on the server side if stream time is used  so we'll need to implement that as a followup